osu.Game/Input/Bindings/DatabasedKeyBindingInputManager.cs

```diff
@@ -51,7 +51,9 @@ private void load(KeyBindingStore keyBindings)
         protected override void Dispose(bool isDisposing)
         {
             base.Dispose(isDisposing);
-            store.KeyBindingChanged -= ReloadMappings;
+
+            if (store != null)

```

Comment:
how can this become null?

Reply:
As discussed, probably a code path down TestCaseReplay that allows it to get finalised before bdl is invoked. Not too sure as I don't have a stacktrace handy atm, but I've had this happen more than once.

Can remove if preferred but...

Reply:
nah, this makes sense